src/provider/client.ts

```diff
@@ -18,7 +19,9 @@ export async function createClient(git: Git, tokens: Tokens, uri: vscode.Uri,
   }
   switch (tokenInfo.provider) {
     case 'github':
-      return new GithubClient(protocol, hostname, tokens[hostname].token, logger);
+      return new GithubClient(
```

Comment:
Please add it for the gitlab client as well.

Reply:
added in new commit